system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -106,9 +106,9 @@
                                (:status-message %)))
                       (:collection-statuses collection-response))))))
 
-    (testing "Data center find and replace"
+    (testing "Data center find and update"
       (let [bulk-update-body {:concept-ids concept-ids
-                              :update-type "FIND_AND_REPLACE"
+                              :update-type "FIND_AND_UPDATE"
                               :update-field "DATA_CENTERS"
                               :find-value {:ShortName "NSID"}
                               :update-value {:ShortName "NSIDC"
```

Comment:
The combination of tests already cover this, which is why I assume the integration test doesn't bother to cover all scenarios as written.  I didn't write the tests so I don't really know why it is so narrow.  I assume you did it this way for a reason?